adot/python/src/otel/otel_sdk/otel-instrument

```diff
@@ -41,7 +41,11 @@ environ["OTEL_RESOURCE_ATTRIBUTES"] = "%s,%s" % (
 )
 
 # xray propagator
-environ["OTEL_PROPAGATORS"] = "aws_xray"
+configured_propagators = environ.get("OTEL_PROPAGATORS")
+if not configured_propagators:
+    environ["OTEL_PROPAGATORS"] = "aws_xray"
```

Comment:
We have a technical debt here to remove otel-instrument in downstream. If you have spare time can play a trick to add additional env variables in downstream. Refer JS example https://github.com/aws-observability/aws-otel-lambda/blob/main/nodejs/scripts/otel-handler

Reply:
Java example might be even clearer since it has the optional env var override too

https://github.com/aws-observability/aws-otel-lambda/blob/main/java/scripts/otel-handler#L5

Reply:
Sounds good! We'll follow up with cleaning up this file as much as possible in a future PR.